front/src/modules/project/components/AddOrEditProjectModal.tsx

```diff
@@ -385,7 +386,9 @@ export default function AddOrEditProjectModal({
                 </div>
               }
               value={
-                currentProject.budget !== undefined && currentProject.budget !== null
+                currentProject.budget !== undefined &&
+                currentProject.budget !== null &&
+                currentProject.budget > 0
```

Comment:
i think this behavior is ok, but maybe we need to double check with a PO

Reply:
`value={currentProject.budget > 0 ? currentProject.budget : ''}` it think do it this way is more simple cause if budget is udefined or null it will also return ''

Reply:
Tried but it doesn't work because budget can be null or undefined and then we can't make the '>0' check.
I added '>=0' because PO said the user wants the see 0 if he types 0